src/lib/components/Tabs/_styles.scss

```diff
@@ -88,6 +88,7 @@ web-tabs {
     display: flex;
     flex: 1;
     flex-direction: column;
+    overflow-y: scroll;
```

Comment:
Do you want it to always be scrollable? Seems like `auto` would show the scrollbar when it's needed.